packages/shi/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "@canonical/shi",
+  "description": "Standard utility functions for Canonical's Web Engineering team",
+  "version": "0.9.0-experimental.12",
+  "type": "module",
+  "module": "dist/esm/index.js",
+  "types": "dist/types/index.d.ts",
+  "files": ["dist"],
+  "author": {
+    "email": "webteam@canonical.com",
+    "name": "Canonical Webteam"
+  },
+  "exports": {
+    ".": {
+      "import": "./dist/esm/index.js",
+      "types": "./dist/types/index.d.ts",
+      "default": "./dist/esm/index.js"
+    }
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/canonical/ds25"
+  },
+  "license": "LGPL-3.0",
+  "bugs": {
+    "url": "https://github.com/canonical/ds25/issues"
+  },
+  "homepage": "https://github.com/canonical/ds25#readme",
+  "scripts": {
+    "build": "tsc -p tsconfig.build.json",
```

Comment:
I'm not 100% familiar with ttl, but is there some way of validating that the code is valid? Is there some build step to increase confidence in code in a given PR? How do we validate its output programmatically?